src/locales/index.js

```diff
@@ -1,9 +1,10 @@
+exports.ar = { common: require("./ar/common.json"), };
```

Comment:
could you remove the 'ar'? i think its outside of the scope of this feature :pray: